assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -58,14 +58,18 @@ const ReplacementContainer = styled.div`
   opacity: ${({ hasReplacement }) => (hasReplacement ? 1 : 0)};
 `;
 
-function DisplayElement({ element, previewMode }) {
+function DisplayElement({ element, previewMode, page }) {
   const {
     actions: { getBox },
   } = useUnits();
-  const {
+  let {
     state: { currentPage },
   } = useStory();
 
+  if (!currentPage) {
+    currentPage = page;
+  }
```

Comment:
This is to allow either a provided page object (from the iterated instances of Dashboard) or use the currentPage as provided by the `useStory` hook/context.

Reply:
I'd rather just pass `page` to this component and do `useStory()` in its caller. Most likely other components always do that anyway.

Reply:
Would you like that change made to the editor side? I can do that here.